gtsfm/data_association/point3d_initializer.py

```diff
@@ -287,15 +289,10 @@ def extract_measurements(self, track: SfmTrack2d) -> Tuple[CameraSetCal3Bundler,
                 track_cameras.append(self.track_camera_dict.get(i))
                 track_measurements.append(uv)
             else:
-                logger.warning("Unestimated cameras found at index {}. Skipping them.".format(i))
+                logger.warning("Unestimated cameras found at index %d. Skipping them.", i)
 
+        # triangulation is impossible with <2 measurements
         if len(track_cameras) < 2 or len(track_measurements) < 2:
-            raise Exception(
```

Comment:
Why is this exception being removed? This class I think is in need of better failure handling. There seem to be multiple failure modes but only cheirality is handled via a Boolean (which is not quite standard in python)

Reply:
Thanks for the review @akshay-krishnan. So un estimated cameras are no longer considered an exception, but expected behavior for some datasets, so here we now handle them gracefully, and ignore the track if we only have 1 estimated camera view for the track. I worry that if we log each one, it could get very verbose over 1000s of tracks.